packages/webpack/src/ember-webpack.ts

```diff
@@ -238,15 +252,19 @@ const Webpack: PackagerConstructor<Options> = class Webpack implements Packager
 
     // loading these lazily here so they never load in non-production builds.
     // The node cache will ensures we only load them once.
-    const [Terser, srcURL] = await Promise.all([import('terser'), import('source-map-url')]);
+    const srcURL = await import('source-map-url');
+    // const [Terser, srcURL] = await Promise.all([import('terser'), import('source-map-url')]);
```

Comment:
I think we can load esbuild here lazily just like we did with terser, so it never even loads in dev builds.

Reply:
Do we want to manage the esbuild version in embroider (for consistent versions between the two minification bits)? If so, that requires the implementation key in the plugin, which requires a regular import 🤷

Reply:
I was assuming we'd only configure the plugin in production builds anyway, so the `await import` and the appending of the minifier plugin could live in the same conditional block. But if there's a reason to keep the plugin configured all the time that's OK.